raiden_contracts/contract_manager.py

```diff
@@ -164,6 +165,16 @@ def merge_deployment_data(dict1: Dict, dict2: Dict) -> Dict:
     return result
 
 
+def version_provides_services(version: Optional[str]) -> bool:
+    if version is None:
+        return True
+    if version == '0.3._':
+        return False
+    if version == '0.8.0_unlimited':
+        return True
+    return compare(version, '0.8.0') > -1
```

Comment:
Hmm this is confusing. What exactly do you want to achieve with the last check.

Looking at the [docs](https://python-semver.readthedocs.io/en/latest/usage.html) of the semver module this will return true for versions >= 0.8.0.

If this is what you want to achieve then perhaps do `compare(version, '0.8.0') >= 0` since this way it will be a bit more obvious what is happening. The `-1` had me confused and I had to check the docs. And perhaps also add a docstring as not everyone knows how this compare function looks like.

Reply:
I'll change to `>= 0`.  I agree it's easier.

Reply:
Thanks man